### PR TITLE
Vibe 2.0 - css modules migration 7

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/4318544025

- [ ] a - https://github.com/mondaycom/monday-ui-react-core/pull/863
  - [ ] 1. Dropdown
  - [ ] 2. SplitButton
  - [ ] 3. Steps
  - [ ] 4. StepsCommand
  - [ ] 5. StepsDot
  - [ ] 6. StepsGalleryHeader
  - [ ] 7. StepsHeader
  - [ ] 8. StepNumbersHeader
- [ ] b - https://github.com/mondaycom/monday-ui-react-core/pull/865
  - [ ] 9. Slider
  - [ ] 10. SliderBase
  - [ ] 11. SliderTrack
  - [ ] 12. SliderFilledTrack
  - [ ] 13. SliderThumb
  - [ ] 14. SliderRail
  - [ ] 15. SliderInfix

### CODE REVIEW

File: src/components/Slider/SliderBase/SliderBase.module.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  padding: 0 8px;
+}
+
+.medium {
+  padding: 0 12px;
+}
+
+.large {
+  padding: 0 12px;
+}
```
Comment (shlomitc) [inline src/components/Slider/SliderBase/SliderBase.module.scss:26]: aren't those should be taken from the spacing tokens?

File: src/components/Dropdown/__tests__/driver.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ export default class DropdownDriver {
   }
 
   get dropdownMenu() {
-    return this.renderResult.container.querySelector(".menu");
+    return this.renderResult.container.querySelector("[class*='-menu']");
```
[the remark this answers is not in the document]
Comment (SergeyRoyt) [reply src/components/Dropdown/__tests__/driver.js:42]: Wasn't able to pass data-testid to the Menu :(

File: src/components/Slider/SliderBase/SliderBase.module.scss (hunk 1 above)
Comment (SergeyRoyt) [reply src/components/Slider/SliderBase/SliderBase.module.scss:26]: Fixed

**Review by shlomitc — APPROVED** [review]
just a minor comment

**Pull request closed without merging**